packages/dcos-integration-test/extra/test_3dt.py

```diff
@@ -161,18 +172,18 @@ def test_3dt_nodes(cluster):
         validate_node(response['nodes'])
 
 
-def test_3dt_nodes_node(cluster):
+def test_3dt_nodes_node(cluster, make_3dt_request):
     """
     test a specific node enpoint /system/health/v1/nodes/<node>
     """
     for master in cluster.masters:
         # get a list of nodes
-        response = make_3dt_request(master, BASE_ENDPOINT_3DT + '/nodes', cluster)
+        response = make_3dt_request(master, '/nodes', False)
```

Comment:
nit: The last parameter being passed to these isn't really obvious what it is / means. Doing `is_agent=False` explicitly for these would help quite a bit.

Could also make the `is_agent` implicit by doing `is_agent = host in cluster.agents` in the 3dt check function